crates/ruff_linter/src/rules/flake8_annotations/rules/definition.rs

```diff
@@ -469,7 +469,7 @@ impl Violation for MissingReturnTypeClassMethod {
 /// ```
 ///
 /// ## References
-/// - [PEP 484](https://www.python.org/dev/peps/pep-0484/#the-any-type)
+/// - [Typing documentation: `Any`](https://typing.readthedocs.io/en/latest/spec/special-types.html#any)
```

Comment:
This more specifically takes you to the part of typing.readthedocs.io that constitutes the technical specification rather than the HOWTO guides. (I think that's correct — I think the spec is what we want here!)

```suggestion
/// - [Typing spec: `Any`](https://typing.readthedocs.io/en/latest/spec/special-types.html#any)
```